crypto/engine/eng_devcrypto.c

```diff
@@ -437,25 +605,24 @@ static int digest_update(EVP_MD_CTX *ctx, const void *data, size_t count)
 
     if (count == 0)
         return 1;
+    if (EVP_MD_CTX_test_flags(ctx, EVP_MD_CTX_FLAG_ONESHOT)) {
+        if (digest_op(digest_ctx, data, count, digest_ctx->digest_res, 0) >= 0)
+            return 1;
+    } else if (digest_op(digest_ctx, data, count, NULL, COP_FLAG_UPDATE) >= 0)
+        return 1;
```

Comment:
I missed this...  when we have braces for one branch if a `if`, we want braces for both branches.

Reply:
I've pushed a commit fixing all these.